x-pack/plugins/triggers_actions_ui/public/application/lib/value_validators.ts

```diff
@@ -31,3 +31,12 @@ export function throwIfIsntContained<T>(
     return values;
   };
 }
+
+const urlExpression = /^https?:\/\/([\w\d\-]+\.)+\w{2,}(\/.+)?$/;
+
+export const isUrlInvalid = (url: string | null | undefined) => {
```

Comment:
A couple of cases this doesn't handle: `http://localhost` (maybe used for testing, or perhaps a service running beside kibana) and `http://abc.com:123` (doesn't accept the port).

I think we should use the built-in URL constructor to test, making sure it doesn't throw an error, and then ensure the resulting `protocol` value is `http` or `https`.